learning_journal/templates/edit.jinja2

```diff
@@ -10,8 +10,10 @@
     {% endfor %}
     </ul>
   {% endif %}
-    <p>{{ field.label }}: {{ field }}</p>
 {% endfor %}
+  <p>{{ form.title.label }}:  {{  form.title }} </p>
+  <p>{{ form.body.label }}: {{ form.body(cols="55", rows="20") }} </p>
+  <p>{{ form.id }} </p>
```

Comment:
I'd probably leave out the `<p>` tag here.  It'll just end up taking up space that doesn't need to be there, since the input is hidden.